### ISSUE TITLE
Will it be released?

### ISSUE THREAD

**Issue by kft334** [magic-research/InstaDrag#2]
I'm only asking because many of your repos have promised code but remain empty to this day and the given time frame for release has passed. Will this actually be released at some point?

**Comment by liewjunhao** [comment]
Yes, we will definitely release the code. Although we have obtained preliminary approval for open-sourcing the code, the code review process was stuck at some point due to internal circumstances. We are sorry for the delay, and we will release it as soon as the code review process completes.

**Comment by kft334** [comment]
Thanks for the update and transparency. Really looking forward to using it when it's made available.

**Comment by liewjunhao** [comment]
Thank you for your patience. Code and pre-trained models are now available [here](https://github.com/magic-research/LightningDrag). Please note that this project is now called [LightningDrag](https://lightning-drag.github.io/) and this repo will be removed soon.
